multi-component-sample/src/main/java/com/what3words/samples/multiple/ui/screen/MainAppScreen.kt

```diff
@@ -75,11 +79,29 @@ fun MainAppScreen(
                 val (w3wTextFieldRef, mapRef, ocrRef, mapTypeRef, addMarkerRef) = createRefs()
 
                 OcrView(
-                    ocrWrapper,
-                    dataProvider,
+                    w3WTextDataSource = w3WTextDataSource,
+                    w3WImageDataSource = w3WImageDataSource,
                     scanScreenVisible = scanScreenVisible,
                     onScanScreenVisibleChange = { scanScreenVisible = it },
-                    onSuggestionScanned = onSuggestionChanged
+                    onSuggestionScanned = { w3wSuggestion ->
+                        val suggestion = SuggestionWithCoordinates(
```

Comment:
@phpduy, can we create a mapping extension for this? W3WSuggestion.ToSuggestionWithCoordinates() so it is less polluted for the clients? We need this mapping because not all our components work with the Core library.